standard/links/commons/links.lua

```diff
@@ -47,6 +47,7 @@ local PREFIXES = {
 		stream = 'https://live.bilibili.com/',
 	},
 	['bilibili-stream'] = {'https://live.bilibili.com/'},
+	bluesky = {'https://bsky.app/'},
```

Comment:
Is this correct prefix?
Should it not be `https://bsky.app/profile`? 
I don't know bluesky very well